library/src/main/java/io/constructor/data/model/AutocompleteResult.kt

```diff
@@ -0,0 +1,4 @@
+package io.constructor.data.model
+
+
+data class AutocompleteResult(val sections: Sections)
```

Comment:
Can we add resultID to this class?

Reply:
What are your thoughts on renaming to `AutocompleteResponse`?

Reply:
yes, sounds better